Features1/tests/features/deleteContact.feature

```diff
@@ -0,0 +1,9 @@
+Feature:Delete Contact
+    As a user
+    I want to delete my contacts 
+    So that i can remove unwanted contacts
+
+    Scenario: Delete contacts
+        Given the user has navigated to the contact page
+        When  the user click "delete" botton
```

Comment:
how would you know which `delete` button to click if there are multiple contacts? may be give user information which one to delete.